pkgs/applications/video/gpu-screen-recorder/default.nix

```diff
@@ -37,16 +43,21 @@ stdenv.mkDerivation {
     wayland
     libdrm
     libva
+    libXdamage
+    libXi
     libXrandr
     libXfixes
   ];
 
-  buildPhase = ''
-    ./build.sh
-  '';
+  mesonFlags = [
+    # FIXME: systemd unit file gets installed to absolute path.
+    "-Dsystemd=false"
+
+    # FIXME: Capabilities can not be set if not running the build with root permissions.
+    "-Dcapabilities=false"
```

Comment:
Would appreciate to have some feedback/help on this, I'm unfortunately too little of a NixOS aficionado to figure this out myself (or if it is even possible).

```
Running custom install script '/build/extra/meson_post_install.sh'
unable to set CAP_SETFCAP effective capability: Operation not permitted
\n!!! Please re-run install as root\n
unable to set CAP_SETFCAP effective capability: Operation not permitted
FAILED: install script '/build/extra/meson_post_install.sh' failed with exit code 1.
```

Reply:
In the case of nix it has to be done differently: https://github.com/NixOS/nixpkgs/issues/275481

Reply:
I'll take a look, thanks for the pointer.

Reply:
Implemented now and seems to work great.

@sersorrel I assigned you a co-author attribution on the respective commit since this is based on your issue report, please let me know in case you want me to omit it or something is wrong with the metadata.